examples/jms-consumer-acknowledgement/jms_consumer_acknowledgement.md

```diff
@@ -0,0 +1,18 @@
+# JMS message producer - Manual acknowledgment
+
+The `jms:MessageConsumer` allows manual acknowledgment for received messages. A `jms:MessageConsumer` can be initialized by using a `jms:Session` object. To configure manual acknowledgment mode,  create a `jms:Session` object with acknowledge mode set to `jms:CLIENT_ACKNOWLEDGE`.
+
+::: code jms_consumer_acknowledgement.bal :::
+
+## Prerequisites
+- Start a [ActiveMQ broker](https://activemq.apache.org/getting-started) instance.
+
+Run the program by executing the following command.
```

Comment:
```suggestion
Run the program by executing the following command.
```
Should this also be under the `Prerequisites` section?

Reply:
Nope